MOcov/mocov_find_files.m

```diff
@@ -67,7 +86,9 @@
         if ~ignore_fn
             path_fn=fullfile(root_dir, fn);
 
-            if isdir(path_fn)
+            if ~isempty(regexp(fn,exclude_re,'once'));
```

Comment:
Currently this would also exclude directories that match the pattern. Is that intended, or would it make more sense to limit this to files (not directories) only?  In any case, can you please add this in the documentation (around line 15)?